lib/timeutils/timeutils-internals.h

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
As a result of the - VERY appreciated - conv.c/h modification this file became futhile. (If I see it correctly, it is already removed from compiling, only the file left in the repo.)